include/micm/solver/backward_euler.inl

```diff
@@ -157,32 +162,27 @@ namespace micm
                       (std::abs(*forcing_iter) <= parameters_.relative_tolerance_ * std::abs(*yn1_iter));
           ++forcing_iter, ++yn1_iter, ++abs_tol_iter;
         } while (converged && forcing_iter != forcing.end());
-
-        if (!converged)
-        {
-          std::cout << "failed to converge within the newton iteration\n";
-        }
       } while (!converged && iterations < max_iter);
 
       if (!converged)
       {
-        std::cout << "failed to converge\n";
+        result.stats_.rejected_++;
         n_successful_integrations = 0;
 
         if (n_convergence_failures >= time_step_reductions.size())
         {
-          // we have failed to converge, accept the solution
-          // TODO: continue on with the current solution to get the full solution
-          n_convergence_failures = 0;
-          // give_up = true;
           t += H;
-          throw std::system_error(make_error_code(MicmBackwardEulerErrc::FailedToConverge), "Failed to converge");
-        };
-
-        H *= time_step_reductions[n_convergence_failures++];
+          std::cerr << "Failed to converge too many times in a row. Accepting the current integration and continuing on.\n";
```

Comment:
I think we should avoid writing to stdout and stderr in favor of letting host applications handle all logging information. Why not throw the FailedToConverge error as before?

Reply:
Camchem assumes we can continue on even when the integration fails. Throwing an error would prevent us from checking our implementation of backward euler against camcehm. I'm not really sure what we should do. What if we add another state to the result enum `AcceptingUnconvergedIntegration`? @sjsprecious @mattldawson

Reply:
You are correct that CAM-Chem can continue the runs even if the solver does not converge for this time step. It will write out a warning message to the log file. Adding another state to the result seems fine to me.